src/main/kotlin/no/nav/syfo/sykmelding/internal/db/SykmeldingDbModel.kt

```diff
@@ -24,3 +24,182 @@ data class SykmeldingDbModel(
     val sykmeldingsDokument: Sykmelding,
     val status: StatusDbModel
 )
+
+data class Sykmelding(
+    val id: String,
+    val msgId: String,
+    val pasientAktoerId: String,
+    val medisinskVurdering: MedisinskVurdering,
+    val skjermesForPasient: Boolean,
+    val arbeidsgiver: Arbeidsgiver,
+    val perioder: List<Periode>,
+    val prognose: Prognose?,
+    val utdypendeOpplysninger: Map<String, Map<String, SporsmalSvar>>,
+    val tiltakArbeidsplassen: String?,
+    val tiltakNAV: String?,
+    val andreTiltak: String?,
+    val meldingTilNAV: MeldingTilNAV?,
+    val meldingTilArbeidsgiver: String?,
+    val kontaktMedPasient: KontaktMedPasient,
+    val behandletTidspunkt: LocalDateTime,
+    val behandler: Behandler,
+    val avsenderSystem: AvsenderSystem,
+    val syketilfelleStartDato: LocalDate?,
+    val signaturDato: LocalDateTime,
+    val navnFastlege: String?
+)
+
+data class MedisinskVurdering(
+    val hovedDiagnose: Diagnose?,
+    val biDiagnoser: List<Diagnose>,
+    val svangerskap: Boolean,
+    val yrkesskade: Boolean,
+    val yrkesskadeDato: LocalDate?,
+    val annenFraversArsak: AnnenFraversArsak?
+)
+
+data class Diagnose(
+    val system: String,
+    val kode: String,
+    val tekst: String?
+)
+
+data class AnnenFraversArsak(
+    val beskrivelse: String?,
+    val grunn: List<AnnenFraverGrunn>
+)
+
+data class Arbeidsgiver(
+    val harArbeidsgiver: HarArbeidsgiver,
+    val navn: String?,
+    val yrkesbetegnelse: String?,
+    val stillingsprosent: Int?
+)
+
+enum class HarArbeidsgiver(val codeValue: String, val text: String, val oid: String = "2.16.578.1.12.4.1.1.8130") {
```

Comment:
Hva er den verdien..?

Reply:
Ah, nå skjønte jeg! Det er den spørsmålsid'en, er det ikke det? Men kanskje vi skulle ha laget konstanter av dem med et beskrivende navn så det blir litt mindre mystisk for nestemann som ser koden?